app/models/team_member.rb

```diff
@@ -4,6 +4,14 @@ class TeamMember < ApplicationRecord
   has_one_attached :profile_image
   has_one_attached :cover_image
 
+  def prev
```

Comment:
Style suggestion: prefer whole words over abbreviations. (`previous` instead of `prev` here)

Reply:
That makes sense. I think I saw someone use prev on stack overflow, which it's safe to assume not always using best practices.